packages/ibm-products/src/components/Coachmark/next/Coachmark/ContentHeader.tsx

```diff
@@ -72,6 +72,25 @@ const ContentHeader = forwardRef<HTMLDivElement, ContentHeaderProps>(
           dragHandle: handleRef.current,
           focusableDragHandle: dragRef.current,
         });
+
+        const onDragStart = () => {
+          if (contentRef) {
+            contentRef.classList.add(`${contentHeaderBlockClass}--is-dragging`);
+            contentRef.setAttribute('aria-label', 'Coachmark is being dragged');
```

Comment:
I think the `ContentHeader` should accept an `aria-label` or some additional prop so that this text has the ability to be translated.